source/user-manual/agents/remote-upgrading/create-custom-wpk/generate-wpk-package-manually.rst

```diff
@@ -185,20 +185,27 @@ Install the root CA if you want to overwrite the root CA with the file you creat
   # cd ../
   # cp path/to/wpk_root.pem etc/wpk_root.pem
 
+Copy the necessary scripts to the current folder to compile the WPK correctly:
+
+.. code-block:: console
+
+  # cp path/to/upgrade.sh .
+  # cp path/to/pkg_installer_mac.sh .
+
 Compile the WPK package using the PKG package and, your SSL certificate and key:
 
 .. code-block:: console
 
-  # tools/agent-upgrade/wpkpack.py output/myagent.wpk path/to/wpkcert.pem path/to/wpkcert.key path/to/wazuhagent.pkg path/to/upgrade.sh path/to/pkg_installer_mac.sh
+  # tools/agent-upgrade/wpkpack.py output/myagent.wpk path/to/wpkcert.pem path/to/wpkcert.key wazuh-agent-|WAZUH_CURRENT_OSX|-|WAZUH_REVISION_OSX|.pkg upgrade.sh pkg_installer_mac.sh
```

Comment:
Removing the path to `path/to/wazuhagent.pkg` means that we should copy this package to the current folder in the previous step as well.